alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
Internal or public, all 'our' projects should be using the same code quality rules. Keeping it explicitly using the same source prevents duplication of work, otherwise if you update a rule locally in this project it would probably make sense to update the other rules. 

It makes sense to have them locally as a reference, but they will always be added to the `app/config/quality` folder anyway when you build with sputnik.